content/cli/workspaces.md

```diff
@@ -0,0 +1,206 @@
+      "entryFile": "main",
+      "sourceRoot": "apps/my-app/src",
+      "compilerOptions": {
+        "tsConfigPath": "apps/my-app/tsconfig.app.json"
+      }
+    }
+  }
+}
+```
+
+### Workspace projects
+
+Workspaces are composed of **projects**.  A project may be either:
+- an **application**: a full Nest application including a `main.ts` file to bootstrap the application.  For almost all intents and purposes, an application within a workspace is functionally identical to a *single project* structure application.
+- a **library**: a library is a way of packaging a general purpose set of features (modules, providers, controllers, etc.) that can be used within other projects.  A library cannot run on its own, and has no `main.ts` file.
+
+All workspaces have a **default application**.  In the `nest-cli.json` file above, the default project is `my-project`.  This is defined by the top-level `"root"` property, which points at the root of the default project.  Usually, this is the *single project* mode application you started with, and later added applications with `nest generate app`.  Default projects are used by `nest` commands like `nest build` and `nest start` when a project name is not supplied.
+
+For example, in the above structure, running
+```bash
+$ nest start
+```
+
+Will start up the `my-project` app.  To start `my-app`, we'd do:
+
+```bash
+$ nest start my-app
+```
+
+#### Workspace global properties
+
+Workspaces have additional workspace-wide properties that help hide the complexity of working in a multi-project environment.  These are reflected in the following top-level properties of the `nest-cli.json` file:
+
+- collection: points at the collection of schematics used to generate components; you generally should not change this value
+- sourceRoot: points at the root of the source code for the *default project*
+- monorepo: for a workspace mode structure, this value is always `true`
+- root: points at the project root of the *default project*
+- compilerOptions: a map with keys specifying compiler options and values specifying the option setting
+
+#### Compiler options
+
+Notice that by default, `compilerOptions` is set as follows:
+```json
+  "compilerOptions": {
+    "webpack": true,
+    "tsConfigPath": "apps/my-project/tsconfig.app.json"
+  }
+```
+
+The `webpack` property, when true (the default), uses [webpack]() to compile and bundle the code.  This is one key difference between *workspace mode* and *single project mode*.  *Single project mode* uses `tsc` for compilation by default.  The reason for this difference is that webpack can have significant advantages in build times and in producing a single file bundling all project components together.  If you wish to generate individual files, set `webpack` to `false`, which will cause the build process to use `tsc`.
+
+**Note to Kamil**: can you please explain the significance of why `compilerOptions.tsConfigPath` is set to the default project?
```

Comment:
> **Note to Kamil**: can you please explain the significance of why `compilerOptions.tsConfigPath` is set to the default project?

That's the path we should look for when someone calls `nest build` without passing the name of the app (like: `nest build NAMEOFTHELIBORAPP`). Same with `nest start`